ipynb/Alpha diversity notebook.ipynb

```diff
@@ -48,7 +48,19 @@
    "metadata": {
     "collapsed": false
    },
-   "outputs": [],
+   "outputs": [
```

Comment:
can you please remove these changes from the PR? doesn't look like they should be in here